src/styles/index.scss

```diff
@@ -1,3 +1,48 @@
+@import './header';
+@import './main';
+
 body {
   margin: 0;
 }
+
+.main {
+  margin: 0 auto;
+  display: grid;
+
+  gap: 46px 48px;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (max-width: 488px) {
+  .main {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
```

Comment:
The grid-template-columns setting here is the same as the default setting outside of media queries. This makes the media query redundant. Consider removing it or adjusting the default setting.